ccp/src/cu/proving_thread/async_/thread.rs

```diff
@@ -166,3 +172,20 @@ impl ProvingThreadFacade for ProvingThreadAsync {
         Ok(())
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct ProvingThreadConfig {
+    pub hashes_per_round: usize,
+    pub async_to_sync_queue_size: usize,
+    pub sync_to_async_queue_size: usize,
+}
+
+impl ProvingThreadConfig {
```

Comment:
Again I'd like to have `From` here b/c it looks more idiomatic

Reply:
I'm not sure that `From` is always a replacement for a single-argument constructor.